coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/functions/GenericTemplateFunction.java

```diff
@@ -58,11 +57,6 @@ public boolean checkOperandTypes(SqlCallBinding callBinding, boolean throwOnFail
     return true;
   }
 
-  @Override
-  protected void checkOperandCount(SqlValidator validator, SqlOperandTypeChecker argTypeChecker, SqlCall call) {
```

Comment:
Not sure if this is safe to remove if it is protected, if we want to ensure compatibility.

Reply:
Added back